tasks/eth/3-add-superchainConfig/justfile

```diff
@@ -4,47 +4,89 @@ export rpcUrl := env_var('ETH_RPC_URL')
 export signatures := env_var_or_default('SIGNATURES', '')
 export bundle := "input"
 export location := `pwd`
-export OWNER_SAFE := env_var('FOUNDATION_SAFE')
 
-simulate hdPath='0':
+# Accounts
+export councilSafe := env_var('COUNCIL_SAFE')
+export foundationSafe := env_var('FOUNDATION_SAFE')
+export ownerSafe := env_var('FOUNDATION_SAFE')
+export randomPersonEoa := "0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045"
+
+simulate whichSafe hdPath='0':
   #!/usr/bin/env bash
   cd ../../..
   echo "getting signer address..."
-  if [ t"$SIMULATE_WITHOUT_LEDGER" == t1 ]
-  then
+  if [ t"{{whichSafe}}" == t"foundation" ]; then
+    safe="${foundationSafe}"
     signer="0x42d27eEA1AD6e22Af6284F609847CB3Cd56B9c64"
-  else
+  fi
+  if [ t"{{whichSafe}}" == t"council" ]; then
+    safe="${councilSafe}"
+    signer="0x07dC0893cAfbF810e3E72505041f2865726Fd073"
```

Comment:
What are the hardcoded signer addresses here? A comment or pulling them into a var would be more clear

Reply:
An env var in the `justfile` makes sense to me. WDYT about deleting `.env` file entirely and having all these values set in the header of this `justfile`?

Reply:
Actually I think we need to keep the `.env` file for the sake of the nonce which is read elsewhere (the forge scripts).

Reply:
Looks like you could remove the `.env` file by having the nonce be another input, and export it from the justfile https://just.systems/man/en/chapter_37.html

> Parameters prefixed with a $ will be exported as environment variables

But I don't feel too strongly here

Reply:
I've discovered that we also have the issue with the `OWNER_SAFE` so I'll keep the .env for now. 

Once we're down to a single justfile in the root I think it's likely we'll want to keep the .env.

Reply:
Fixed: ae4503e843244a6050e4c7092bf3faa14c7f5417